### PR TITLE
I heard you like comments

### PR DESCRIPTION
**Pull request by loopj**

### CODE REVIEW

File: src/main/groovy/com/bugsnag/android/gradle/BugsnagPlugin.groovy [hunk 1]
```diff
@@ -20,41 +34,74 @@ class BugsnagPlugin implements Plugin<Project> {
...
 
                 // Create a Bugsnag task to add a build UUID to AndroidManifest.xml
-                BugsnagManifestTask manifestTask = project.tasks.create("processBugsnag${variant.name.capitalize()}Manifest", BugsnagManifestTask)
+                // This task must be called after "process${variantName}Manifest", since it
+                // requires that an AndroidManifest.xml exists in `build/intermediates`.
+                BugsnagManifestTask manifestTask = project.tasks.create("processBugsnag${variantName}Manifest", BugsnagManifestTask)
                 manifestTask.group = GROUP_NAME
                 manifestTask.manifestPath = manifestPath
-                manifestTask.dependsOn variantOutput.processManifest
+                manifestTask.mustRunAfter variantOutput.processManifest
```
Comment (kattrali) [inline src/main/groovy/com/bugsnag/android/gradle/BugsnagPlugin.groovy]: What is the difference between `dependsOn` and `mustRunAfter`?

Comment (loopj) [inline src/main/groovy/com/bugsnag/android/gradle/BugsnagPlugin.groovy]: `dependsOn` - go ahead and add this task as a dependency (adds the task to be run if it wasn't already)
`mustRunAfter` - if both of these tasks are told to execute, make sure X runs after Y (doesn't actually add a dependency)

Comment (loopj) [inline src/main/groovy/com/bugsnag/android/gradle/BugsnagPlugin.groovy]: https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:adding_dependencies_to_tasks
https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:ordering_tasks

Comment (kattrali) [inline src/main/groovy/com/bugsnag/android/gradle/BugsnagPlugin.groovy]: Gotcha. Looks good. :+1:

**Comment by kattrali** [comment]
:tada:

**Pull request merged**